### PR TITLE
Add missing info population in the activity info. (#936)

### PR DESCRIPTION
**Pull request by luowenking1**

### CODE REVIEW

File: internal/internal_event_handlers.go [hunk 1]
```diff
@@ -494,11 +494,12 @@ func (wc *workflowEnvironmentImpl) ExecuteLocalActivity(params executeLocalActiv
...
-		params:      &params,
-		callback:    callback,
-		retryPolicy: params.RetryPolicy,
-		attempt:     params.Attempt,
+		activityID:   activityID,
+		params:       &params,
+		callback:     callback,
+		retryPolicy:  params.RetryPolicy,
+		attempt:      params.Attempt,
+		workflowTask: &workflowTask{},
```
Comment (venkat1109) [inline internal/internal_event_handlers.go]: why is this needed ? localActivityTask shouldn't and doesn't need a non-nil workflowTask

File: internal/internal_task_pollers.go [hunk 2]
```diff
@@ -464,15 +464,44 @@ func (lath *localActivityTaskHandler) executeLocalActivityTask(task *localActivi
...
+		scheduled = time.Unix(0, task.workflowTask.task.GetScheduledTimestamp())
+	}
+
+	if task.workflowTask != nil && task.workflowTask.task != nil {
+		started = time.Unix(0, task.workflowTask.task.GetStartedTimestamp())
+	}
+
+	if task.workflowTask != nil && task.workflowTask.task != nil {
+		taskToken = task.workflowTask.task.TaskToken
+	}
```
Comment (venkat1109) [inline internal/internal_task_pollers.go]: this doesn't make sense to me. For local activity task, these things don't really exist or mean anything

File: internal/internal_task_pollers.go [hunk 3]
```diff
@@ -465,6 +465,9 @@ func (lath *localActivityTaskHandler) executeLocalActivityTask(task *localActivi
 	}
 
 	ctx := context.WithValue(rootCtx, activityEnvContextKey, &activityEnvironment{
+		workflowType:      &task.params.WorkflowInfo.WorkflowType,
```
Comment (venkat1109) [inline internal/internal_task_pollers.go]: just to be defensive, consider making a copy in a local var and then pass the addr of that local var here

File: internal/internal_workflow_testsuite.go [hunk 4]
```diff
@@ -534,6 +534,38 @@ func (env *testWorkflowEnvironmentImpl) executeActivity(
 	}
 }
 
+func (env *testWorkflowEnvironmentImpl) executeLocalActivityParams(
```
Comment (venkat1109) [inline internal/internal_workflow_testsuite.go]: what is the difference between this and executeLocalActivity ? Why do you need to return concrete type here ?Given that this is unit test, can you type case inside of your test inside of creating this additional method which does the same thing ?

**Comment by luowenking1** [comment]
> Can you please add a test here for local activity :
> https://github.com/uber-go/cadence-client/blob/master/test/integration_test.go
> 
> Inside of your local activity function, assert that the context has the value for the workflowType

I have added the checking there but I did not create a new method. Instead I added the checking in an existence of a local activity testing.
